x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -456,7 +456,7 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
-    it('should prioritize tasks which are called using runNow', async () => {
+    it('should prioritize tasks which are called using runSoon', async () => {
```

Comment:
Actually i am not sure if it prioritize/should prioritize a task. We just set a new `runAt` time.
If there is already a task with earlier runAt, it would be run first...

Reply:
Yes, it's possible that it won't be the next task that is run if there are other tasks that are overdue.

Reply:
Discussed this with Mike and decided to remove the test, as runSoon does not prioritise a task anymore.